app/models/folio/checkout.rb

```diff
@@ -217,16 +217,16 @@ def renewal_count
       record.dig('details', 'renewalCount') || 0
     end
 
-    # nil means "unlimited" for unseenRenewalsRemaining
     def unseen_renewals_remaining
-      Float::INFINITY
-      # fields['unseenRenewalsRemaining'] || Float::INFINITY
+      unseen_renewals_allowed - renewal_count
+    end
+
+    def unseen_renewals_allowed
+      record.dig('details', 'loanPolicy', 'renewalsPolicy', 'numberAllowed') || Float::INFINITY
     end
 
-    # nil means "unlimited" for seenRenewalsRemaining
     def seen_renewals_remaining
-      Float::INFINITY
-      # fields['seenRenewalsRemaining'] || Float::INFINITY
+      0
```

Comment:
The ticket suggests that `seen_renewals_remaining` should be set to 0, but I don't know what seen vs. unseen means so I don't know how to assess whether this is correct.

Reply:
We think Infinity is the right default, because: https://github.com/sul-dlss/mylibrary/blob/main/app/models/folio/checkout.rb#L81